http-server-tck/src/main/java/io/micronaut/http/server/tck/tests/ResponseStatusTest.java

```diff
@@ -48,7 +48,7 @@ public class ResponseStatusTest {
     @Test
     void testConstraintViolationCauses400() throws IOException {
         asserts(SPEC_NAME,
-            HttpRequest.POST("/response-status/constraint-violation", Collections.emptyMap()).header(HttpHeaders.CONTENT_TYPE, MediaType.TEXT_PLAIN),
```

Comment:
breaking change. the text/plain codec cannot serialize a map. previously it would fall back to json i think, now it will fail. the body is unused in the test anyway.